policy/release/tasks_test.rego

```diff
@@ -16,19 +16,21 @@ test_no_tasks_present if {
 		"buildType": lib.pipelinerun_att_build_types[0],
 		"buildConfig": {"tasks": []},
 	}}}]
+
+	lib.assert_equal_results(deny, expected) with input.attestations as _slsav1_attestations_with_tasks([], [])
 }
 
 test_failed_tasks if {
 	expected := {
 		{
 			"code": "tasks.successful_pipeline_tasks",
-			"msg": "Pipeline task \"av-scanner-p\" did not complete successfully, \"Failed\"",
-			"term": "av-scanner-p",
+			"msg": "Pipeline task \"av-scanner\" did not complete successfully, \"Failed\"",
```

Comment:
Why not use the pipeline task name in the error message?

Reply:
I did change that I guess out of convenience. The rule that it tests was printing `task.name` which doesn't exist in slsav1.0 and we already have a helper that pulls the task name from `task.ref.name` which I switched to. Shoudln't be hard to add a helper for the pipeline task name also. Here's the code. https://github.com/enterprise-contract/ec-policies/pull/709/files#diff-e9948535fcfae404a22da4f7fc963cff6fa6faa48fd26a6df42f5f073c6eb3edR71

Reply:
If it doesn't make sense in 1.0, then let's change it. The important thing is that we are always evaluating against the `ref.name`. The error message could display either. Which one is better is subjective.